### ISSUE TITLE
Static pkgs

### ISSUE THREAD

**Issue by hovercats** [hovercats/kiss-dumpsterfire#2]
Here is the list of pkg that currently builds statically in this repo,
aswell as some notes, as to why some pkgs are not
This also serves as my todo list
This list is probably not complete, but gives you a general idea of the state of
static building

### Core:

[x] busybox
[x] bsd-m4
[x] binutils
	- exists in extra, because I use `llvm`
[x] byacc
[x] curl
[x] flex
[x] make (gmake)
[x] git
[x] gnu-m4
	- exists in extra
[x] pigz

[] gcc
	- exists in extra, because I use `llvm` instead, atm anyway.
	- Im still working some stuff out
	- builds static libs
	- does not build statis bins
[] libressl
	- no issue building statically
	- however, python will give you issues due to libssl.so missing
	- see #1
[] llvm
	- still working some stuff out
[] musl
	- builds perfectly fine statically
	- but will break most of your system because of $pkg is linked to libc.so
[] zlib
	- mostly the same as musl
	- needs to build a static toolchain that uses libz.a instead
	- building zlib with just static libs will break, both llvm and gcc
		- aswell as more

- `baselayout` and `baseinit` is not neccessary for me to include
- `grub` is not included, because I dont use it. 
	- I use efistub instead

#### Extra
[x] abduco
[x] axel
[x] bison
[x] brightnessctl
[x] catgirl
[x] dash
[x] devour
[x] dmenu
[x] dvtm
[x] eiwd
[x] expat
[x] fzf
[x] github-cli
[x] gnupg1
[x] lf
[x] links2
[x] lynx
[x] mksh
[x] mtm
[x] ncurses
[x] netbsd-curses
[x] oksh
[x] onetrueawk
[x] openssh
[x] sdhcp
[x] sfeed
[x] st
[x] tabbed
[x] tinyalsa
[x] vim
[x] wpa_supplicant
[x] zsh

[] alsa-lib
	- will probably vendor inside alsautils
[] alsa-utils
	- breaks because of libasound.a is missing
	- need to force alsa-lib to produce libasound.a, instead of .so lib
[] cairo
	- havent looked into it yet
[] cmake
	- needs more looking into
[] dhcpcd
	- udev.so exists, so using -static flag will fail because of said lib
	- need to force udev.a in any case
[] ffmpeg
	- yet to be looked into
[] mesa
	- needs more looking into
	- just needs to produce static libs, which will solve some issues.
[] mpv
	- havent really looked into it.
	- uses waf. not familiar with it atm
[] surf
	- needs mesa to produce static libs
[] python
	- see #1
[] vimb
	- basically same as surf
[] webkit2gtk

#### xorg

[x] glazier
[x] monsterwm
[x] no-wm
[x] sowm
[x] sxhkd
[x] xdotool
[x] xsel

[] feh
[] hsetroot
[] wmutils
[] wmutils-opt
[] xorg-server
	- needs to build some of the mesa libs statically
